### DISCUSSION TITLE
shokaX的tab标签卡没有下划线

### DISCUSSION THREAD

**Discussion by FidStyle** [theme-shoka-x/hexo-theme-shokaX#204] [Q&A] [answered]
shokaX：
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/44d6af47-e5f0-4d95-a167-818bce2ddd60)

shoka：
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/1de45e26-ac72-4bd2-9d29-9b9efc55d47a)

对比可以发现shokaX没有下划线作为表示，读者不知道这是一个可选项。

markdown-it-container相关。

**Comment by FidStyle** [comment]
@zkz098 @D-Sketon

**Reply by zkz098** [reply, marked as answer]
属于Bug，tabs的CSS丢失了left和right属性，且圆角半径不正确，下一个版本修复

**Comment by FidStyle** [comment]
还有个问题，也是css相关（）

[测试网页](https://shokax.joytion.cn/2024/01/26/test/)

在我逐渐缩小宽度时，canvas的width会随之变化直到565px，随后在565到566px之间跳动，宽度从141到140px时，canvas的width从565变为560px，随后递减，奇怪。

会导致手机端的文章页面可拖动，文章宽度与导航栏宽度不一致。

网站主页是没问题的，但是进入文章后会出现。

@zkz098

**Reply by zkz098** [reply]
怪问题
盲猜上游mouse-firework出了问题，我去看看

**Reply by zkz098** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/92242020/72355c90-d1d5-4e7f-a082-742f3b2649fd)
等我发个PR（）

**Reply by zkz098** [reply]
https://github.com/D-Sketon/mouse-firework/pull/1

**Reply by FidStyle** [reply]
已安装 mouse-firework@0.0.5 release版本，没有有效解决问题。我再看看怎么个事（）

**Reply by FidStyle** [reply]
我删除了mouse-firework文件夹并重新hexo cl && g && s 之后仍可复现

**Reply by FidStyle** [reply]
@zkz098

**Reply by FidStyle** [reply]
感觉不太像是windows.innerWidth的问题，虽然确实有关

**Comment by FidStyle** [comment]
已确定原因：开启评论系统可复现，我使用的是waline

**Reply by FidStyle** [reply]
感觉像是评论没做好界面兼容，具体原因待寻找
@zkz098

**Reply by FidStyle** [reply]
测试：关闭recent最新评论后问题仍然存在

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/40ff10ae-bee9-4e62-b40d-67548afd59aa)
罪魁祸首（）

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/a2567545-1327-4086-a9cd-0b7fe3e35b89)
waline的button个数过多溢出导致的，应该是个小bug

**Reply by FidStyle** [reply]
## 我的心情be like 
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/c35508bb-beda-4dde-9d56-f60674ae0004)

**Reply by zkz098** [reply]
cc @D-Sketon

**Reply by FidStyle** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/5a3167dc-cfc3-4b44-97d4-779ae4134e9b)
看能不能让宽度自适应

**Reply by FidStyle** [reply]
应该是waline的问题 @waline/client@2.15.8/dist/waline.css

**Reply by FidStyle** [reply]
解决办法：定位到wl-tabs，取消white-space的nowrap即可
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/c9a7033f-3ea5-47c4-9c2c-30f24d060014)

**Reply by FidStyle** [reply]
已提交Issue https://github.com/walinejs/waline/issues/2295

**Reply by FidStyle** [reply]
找到原因了，待pr通过

**Comment by FidStyle** [comment]
pr已通过，等待下一次release

**Reply by FidStyle** [reply]
https://github.com/walinejs/waline/pull/2297

wl-emoji-popup 这项增加 width:100% 即可解决，无其他影响

@D-Sketon
